tests/Integration/Kernel.php

```diff
@@ -41,20 +41,25 @@ public function getLogDir(): string
         return '/tmp/' . $this->uniqueId . '/log';
     }
 
-    public function registerBundles()
+    public function registerBundles(): iterable
     {
         return [
             new FrameworkBundle(),
             new MareinStandardHeadersCsrfBundle()
         ];
     }
 
-    protected function configureRoutes(RouteCollectionBuilder $routeCollectionBuilder)
+    protected function configureRoutes(RoutingConfigurator $routingConfigurator)
     {
-        $routeCollectionBuilder->add('/', 'kernel::defaultAction', 'index');
-        $routeCollectionBuilder->add('/api/users', 'kernel::defaultAction', 'api_users');
-        $routeCollectionBuilder->add('/user/logout', 'kernel::defaultAction', 'user_logout');
-        $routeCollectionBuilder->add('/user/profile', 'kernel::defaultAction', 'user_login');
+        $routingConfigurator
+            ->add('index', '/')
+            ->controller('kernel::defaultAction')
+            ->add('api_users', '/api/users')
+            ->controller('kernel::defaultAction')
+            ->add('user_logout', '/user/logout')
+            ->controller('kernel::defaultAction')
+            ->add('user_login', '/user/profile')
```

Comment:
Not related, but could be fixed within this pull request. `user_login` and `/user/profile` doesn't match and is confusing. Should be `user_profile`.